Cargo.lock

```diff
@@ -27,7 +27,7 @@ version = "0.19.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a76fd60b23679b7d19bd066031410fb7e458ccc5e958eb5c325888ce4baedc97"
 dependencies = [
```

Comment:
Fix lock pls, it dumps versions

Reply:
tried `cargo update` but nothing changed

Reply:
just merge master, copy-paste lock from master and run build - everything required will be bumped